src/base.jl

```diff
@@ -657,6 +692,26 @@ function get_components_by_name(
     return components
 end
 
+"""
+    get_components_forecasts(
```

Comment:
nit:  Re-copy the function declaration on top of the docstring.  The docstring function has an extra 's' and the alignment is off by one space.